mmf/models/vilbert.py

```diff
@@ -1226,6 +1230,14 @@ def forward(
                 prediction_scores_t.view(-1, self.vocab_size), masked_lm_labels.view(-1)
             )
             output["masked_lm_loss"] = masked_lm_loss.unsqueeze(0)
+        
+        if itm_loss is not False:
+            itm_head = ITM({"type": "itm", "hidden_size": self.vocab_size})
```

Comment:
This should be initialized in the init not in the forward pass.

Reply:
Hi! corrected the initialization thing. I checked the snippet I added separately in similar manner to "ITM head test" but was not able to test the ViLBERTForPretraining (As after loading yaml file its throwing 'dict' object has no attribute 'bert_model_name') even without making changes in original code.